sympy/combinatorics/fp_groups.py

```diff
@@ -378,9 +464,9 @@ def scan_and_fill_f(self, alpha, word):
     def look_ahead(self):
         R = self.fp_group.relators()
         p = self.p
-        for beta in self.omega:
-            # complete scan all relators under all cosets(obviously live)
-            # without making new definitions
+        # complete scan all relators under all cosets(obviously live)
+        # without making new definitions
+        for beta, w in self.omega:
```

Comment:
No that shouldn't be here. Its a wrong change. I was trying to use 
`product` here, and realized that there is to be a use of `continue with β`.

On 06/18/2016 04:06 PM, Kalevi Suominen wrote:
>
> In sympy/combinatorics/fp_groups.py 
> <https://github.com/sympy/sympy/pull/11231#discussion_r67599158>:
>
> > @@ -378,9 +464,9 @@ def scan_and_fill_f(self, alpha, word):
> >      def look_ahead(self):
> >          R = self.fp_group.relators()
> >          p = self.p
> > -        for beta in self.omega:
> > -            # complete scan all relators under all cosets(obviously live)
> > -            # without making new definitions
> > +        # complete scan all relators under all cosets(obviously live)
> > +        # without making new definitions
> > +        for beta, w in self.omega:
>
> Should |w| really be here?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub 
> <https://github.com/sympy/sympy/pull/11231/files/cf9ce8dfab3b29154db79c917b8e733148045e2c#r67599158>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AHWt8UbGvW6Uzbh3uwF_xPRe3RI0epZSks5qM8oagaJpZM4IzwKC>.
>